vault/dendron.topic.links.child-link.md

```diff
@@ -0,0 +1,39 @@
+---
+id: 4ah3xgz20oylzu3v5segyge
+title: Child Link
+desc: ''
+updated: 1659080332215
+created: 1658828670537
+---
+
+## Summary
+
+![Children](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/Quickstart_-_Dendron.jpg)
```

Comment:
this image isn't great (it doesn't work with dark theme and looks different. can we do a new image using the preview instead?)